manage-apps/register.html

```diff
@@ -204,9 +206,15 @@
           $('#sandboxRegistration button[type="submit"]').attr('disabled', true);
         },
         success: function(data) {
-          console.log(data);
-          $("#sandboxRegistration").addClass('submited');
-          $('#form-success').removeClass('form-success-hidden');
+          if (data.sandboxErr !== 'No Error') {
```

Comment:
The problem is, the HTTP status code that comes back from the sandbox call is a 200 OK, even though there is an error in the body. I could manufacture a status code I guess, and pass that back?